project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Instagram icon is invisible (it's white, same as the background)
<img width="278" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/84dd6b72-6af3-49b9-9c0f-49670fe256c0">